.github/workflows/promote-playground.yml

```diff
@@ -1,13 +1,17 @@
-name: Promote playground on tag
+name: Promote playground on release
 
 on:
-  # Automatically trigger after Release workflow completes
+  # Automatically trigger after Release workflow completes on master
   workflow_run:
     workflows: ["Release"]
     types:
       - completed
+    branches:
+      - master
   # Optional: Allow manual triggering (will auto-detect latest tag)
   workflow_dispatch:
+    branches:
+      - master
```

Comment:
Shouldn't this just happen on any tag push?

```
on:
  push:
    tags:
      - '**'   # matches ALL tags, even with slashes
```

iirc the "Release" workflow runs on any master commit

Reply:
the problem with tag push is that tags created by the Release workflow using GITHUB_TOKEN don't trigger the 'on.push.tags' event due to GitHub security policy (prevents infinite workflow loops), so i thought about an alternative that could be react on the Release workflow completed but only on master, because that is called only when new releases are performed (and tags are added). wdyt?